nanvm-lib/tests/test.f.mjs

```diff
@@ -23,6 +23,7 @@ export default {
             },
             nullish: () => {
                 n(false)(undefined)
+                n(false)(null)
```

Comment:
Add the same operations into the JS file.

Reply:
I am ready to do so but I need instructions on how to run this JS in a local enlistment.

Reply:
`npm test` will run all tests. The log is huge so I run `npm test > _.txt` sometimes.